app/src/main/java/edu/uc/hickmadc/customnotifications/MainActivity.kt

```diff
@@ -42,6 +38,9 @@ class MainActivity : AppCompatActivity() {
         setupBottomNavigationBar()
     }
 
+    /**
+     * Sets up the navigation bar on the bottom of the screen.
+     */
```

Comment:
Added javadoc.

Reply:
While JavaDoc is a good idea, it's not always necessary on private methods, if the method name is self-descriptive.